lib/Conversion/TorchToTosa/TorchToTosa.cpp

```diff
@@ -635,6 +635,56 @@ class ConvertAtenSqueezeAllDimsOp : public ConvertAtenSqueezeOp<AtenOpT> {
   }
 };
 
+// FIXME(AG): This will eventually go into a Tosa*Utils file
+// Convert an fp32 scalar into tosa fp32 tensor.
+static LogicalResult tosaF32TensorFromTorchFloat(ConversionPatternRewriter &rewriter,
+                                        Operation *op, Value torchScalarValue,
+                                        Value &tosaTensor) {
+  double scalarValue;
+  // Check if this is an unrealized conversion cast
+  auto uConvCast = dyn_cast<mlir::UnrealizedConversionCastOp>(torchScalarValue.getDefiningOp());
```

Comment:
Hmm, why do we need to check `UnrealizedConversionCastOp` here? Do you have the IR dump?

Reply:
If you pass the MLIR in basic.mlir for '@torch.aten.pow.Tensor_Scalar', then it leads to an UnrealizedConversionCastOp which has the value attached to it. I do not have the IR dump.

Reply:
oh, I think I understand now. There is no direct conversion from TorchType to TosaType so the `UnrealizedConversionCastOp` was inserted. 

However, to get the original unconverted Torch::FloatType scalar  you can use `op.exponent()` instead of using `adaptor.exponent();`. The operands got from adaptor are converted types and the operands got from the original Torch op are original torch types.

Reply:
Thanks, works.